### PR TITLE
update Discard reasons, make confirm button save changes

### PR DESCRIPTION
**Pull request by haoxusci**
1. add Unspecific into discard reason list, and re-order the list
2. Confirm button works both for confirming the suggested annotation and save annotations if any changes

### CODE REVIEW

File: src/SingleCellAnnotator.imjoy.html [hunk 1]
```diff
@@ -1437,7 +1438,7 @@
           </div>
             <br>
           <div class="btn-group">
-            <button :disabled="current_image_data.location_changed || current_image_data.discarded || current_image_data.unsure" class="btn transparent-btn" @click="confirmAnnotation()">&nbsp;Confirm</button>
+            <button :disabled="current_image_data.discarded || current_image_data.unsure" class="btn transparent-btn" @click="confirmAnnotation()">&nbsp;Confirm</button>
```
Comment (oeway) [inline src/SingleCellAnnotator.imjoy.html]: Should we change the Confirm to Save to be consistent with the save button in segmentation? This button will not only confirm unchanged labels, but also force save existing changed labels.

Comment (haoxusci) [reply src/SingleCellAnnotator.imjoy.html]: sure

Comment (haoxusci) [reply src/SingleCellAnnotator.imjoy.html]: changed the button name and func name accordingly

File: src/SingleCellAnnotator.imjoy.html [hunk 2]
```diff
@@ -955,10 +954,12 @@
...
-          "negative",
-          "out of focus", 
-          "artifact",
-          "missing channels",
+          "Negative",
+          "Unspecific",
+          "Missing channels",
+          "Cell artifacts",
+          "Out of focus", 
+          "Other",
```
Comment (oeway) [inline src/SingleCellAnnotator.imjoy.html]: Remove other and add “Bad segmentation”.

Comment (haoxusci) [reply src/SingleCellAnnotator.imjoy.html]: Changed

**Review by oeway — APPROVED** [review]

**Pull request merged**